src/app/shared/components/comment-form/comment-form.component.ts

```diff
@@ -34,9 +33,11 @@ export class CommentFormComponent implements OnInit {
   loading = false;
 
   commentForm = new FormGroup({
+    type: new FormControl('comment'),
     content: new FormControl(),
   });
 
+  minDate = new Date();
```

Comment:
I think we should prevent the user from submitting a warning with an `exposedUntilDate` set to a past date.

Reply:
👍